randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -681,7 +687,7 @@ Extra - room_id: [17]
   * L0 Hatch to Aquarium Shaft West/Door to Aquarium Tank
   * Extra - door_idx: (39,)
   > Pickup (Missile Tank)
-      Can Use Power Bombs
+      Can Kill Tough Beam-Resistant Enemy and Can Use Power Bombs
```

Comment:
IDK if this requirement should be here. its very easy to just fall down after laying the pb and instantly collect the item without needing to interact with the pirate.